core/hotspot/traffic_shaping.go

```diff
@@ -86,13 +86,13 @@ func (c *baseTrafficShapingController) performCheckingForConcurrencyMetric(arg i
 		if concurrency <= specificConcurrency {
 			return nil
 		}
-		return base.NewTokenResultBlocked(base.BlockTypeHotSpotParamFlow, fmt.Sprintf("Frequency params traffic shaping controller, current concurrency: %d, specific concurrency: %d", concurrency, specificConcurrency))
+		return base.NewTokenResultBlocked(base.BlockTypeHotSpotParamFlow, fmt.Sprintf("Hotspot parameters traffic shaping controller, current concurrency: %d, specific concurrency: %d", concurrency, specificConcurrency))
```

Comment:
Actually we could carry the rule and the triggered value in `BlockError` (using `NewTokenResultBlockedWithCause`), rather than in the unstructured message. We may improve this later.

Reply:
Updated